### DISCUSSION TITLE
Validate built links before publishing

### DISCUSSION THREAD

**Discussion by louij2** [jef/streetmerchant#985] [Ideas and feature requests]
The link under 6800 and 6800xt under series is down so it forwards to the generic AMD GPU page and then alerts saying that 5600xt/last generation cards are 6800/xt 

![image](https://user-images.githubusercontent.com/2940588/100484724-78fe8900-30f5-11eb-9350-00f9fb544ab6.png)

```
			{
					series: 'rx6800',
					url: 'https://www.scan.co.uk/shop/computer-hardware/gpu-amd/amd-radeon-rx-6800-pcie-40-graphics-cards'
			},
			{
					series: 'rx6800xt',
					url: 'https://www.scan.co.uk/shop/computer-hardware/gpu-amd/amd-radeon-rx-6800-xt-pcie-40-graphics-cards'
			}
```

**Comment by jef** [comment]
Unfortunately, we can't do much about this as of now. Going to move to discussions so that potentially be picked up for more validation before building links.
